packages/bsky/src/data-plane/bsync/index.ts

```diff
@@ -210,6 +221,61 @@ const handleNotificationPreferencesOperation = async (
   return handleGenericOperation(db, req, now)
 }
 
+const handleSubjectActivitySubscriptionOperation = async (
```

Comment:
I found the control flow in here a little difficult to follow, i.e. interactions between `method`, `enabled`, early return vs. falling through.